### DISCUSSION TITLE
p/2b868e0020e640bb997c935ae8453bcd/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#25] [Announcements]
# p/2b868e0020e640bb997c935ae8453bcd/

这周的周报晚了一天，因为这两天头疼和发炎，吾辈怀疑已经重阳了，但还没去测试，无论如何，公司内已经有两个了。  开发上周公司的事情很多，web chat 改造和 langchain 一直没有去看，这周还是想抽时间看看。下周五一就回广州了，或许可以和那边的朋友交流一些 gpt 应用的想法。 之前设计的强类型的数据查询 API 被组内吐槽太过复杂，想到了关于技术的使用门槛的事情，所以也想说一下。在一个社

https://blog.rxliuli.com/p/2b868e0020e640bb997c935ae8453bcd/

**Comment by enpitsuLin** [comment]
prisma其实可以完全不用了解他那个DSL怎么用，可以[拉取数据库字段自动生成](https://www.prisma.io/docs/reference/api-reference/command-reference#db-pull)来着，而且其实也不是很难。

**Reply by rxliuli** [reply]
额外的工具链和复杂性不适合第一次尝试的人，开箱即用然后渐进增强，这可能是更好的

**Comment by miaopass-future** [comment]
NodeJS又开始流行做后端语言了么

**Reply by rxliuli** [reply]
主要是上了 docker 之后不用管环境之类的东西很舒服，对于部署而言语言不再重要了
